src/utils/pegRpcProviderByModulus.ts

```diff
@@ -0,0 +1,27 @@
+export const pegRpcProviderByModulus = (request: Request): string | undefined => {
+	let peggedRpcProvider;
+
+	try {
+		const RPC_PROVIDERS_ARR = ['helius', 'triton'];
+
+		const { headers } = request;
+		const clientIdentifier = headers.get('Client-Identifier'); // The header key will need to align with what client applications provide
+
+		// TODO: func to transform clientIdentifier to number type based on clientIdentifier format (e.g., uuid)
+
+		// Only peg to RPC provider if Client-Identifer:
+		// - is provided by client application
+		// - is of number type (so modulus operation won't throw)
+    // CURRRENTLY THIS BLOCK WILL NEVER BE ENTERED 
+		if (clientIdentifier && typeof clientIdentifier === 'number') {
```

Comment:
Won't a header value always be a string? We should just make client identifier any string, then hash the string and take a modulus on the hash

Reply:
I really didn't want the conditional to be entered until we're ready so I added the `typeof` check